docs/igor0806/GamesProject/styles/calculator.css

```diff
@@ -0,0 +1,110 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body {
```

Comment:
Proly you should make one styles file for comment styles and put there for example styles for `body` and then connect it all pages.
It would be better, than simply copy-paste all these `.container` `.intro`, `.nav`, `.footer`...